Makefile

```diff
@@ -24,20 +24,28 @@ deps:
 checks:
 	./scripts/checks "$(PKGS)"
 
-build: deps checks
+run-build: deps checks
 	godep go install -v $(TO_BUILD)
 
+build: start
+	vagrant ssh netplugin-node1 -c 'sudo -i bash -lc "source /etc/profile.d/envvar.sh && cd /opt/golang/src/github.com/contiv/netplugin && make run-build"'
```

Comment:
ok, but if user were to just do `make build` and stop. They won't know that we left a vm running behind. And If they decide to yank out their workspace and pull a new one, this vm will stay there consuming resources. 

I might be coming out a bit paranoid but cleanup somewhat seems a lot safer at expense of spending some time :)